app/presenters/brexit_taxons_presenter.rb

```diff
@@ -0,0 +1,42 @@
+class BrexitTaxonsPresenter
+  FEATURED_TAXONS = %w(
+    /going-and-being-abroad
+    /crime-justice-and-law
+    /transport
+    /environment
+    /business-and-industry
+    /education
+  ).freeze
+
+  def featured_taxons
+    @featured_taxons ||= FEATURED_TAXONS
+      .map { |base_path| Taxon.find(base_path) }
+      .each.with_index(1).map { |taxon, index| BrexitTaxonPresenter.new(taxon, index) }
+  end
+
+  def other_taxons
+    @other_taxons ||= ContentItem.find!('/')
+      .linked_items('level_one_taxons')
+      .reject { |content_item| FEATURED_TAXONS.include?(content_item.base_path) }
+      .reject { |content_item| content_item.base_path == "/government/all" }
+      .select { |content_item| search_response_including_brexit(content_item.content_id, document_types).total.positive? }
+      .map { |content_item| Taxon.find(content_item.base_path) }
```

Comment:
We can look at the links for the campaign and get all the finders that are tagged to it.

That way we only need to make one request for a content item, the campaign, and no longer need to request the home page content item.

I think we should also say `FEATURED_FINDERS`. Our campaign is featuring "Prepare EU Exit" finders, the knowledge of how they related to taxons belongs in the publishing app rummager.

Reply:
Discussed the `FEATURED_FINDERS` with @andrewgarner. I think for now we're going to keep as-is, because on this page `FEATURED_TAXONS` are not finders and this page could be changed so that the links point to something other than finders.

Reply:
As for @sihugh proposal about the base path and title of the taxon, happy to do this - do you want it done now or at some point later (to avoid that blocking merging)?